docs/Developers/Git_Basics.rst

```diff
@@ -70,7 +70,10 @@ Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
```

Comment:
@jayvdb To clarify: You would like to see `git status` , `git remotes` , and `master` all explained under the "Getting to work" section?